canonical/ubuntu.com
# WD-12061 - rebrand /cloud/multi-cloud

## Done

Rebrand /cloud/multi-cloud. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14082.demos.haus/cloud/multi-cloud)
- [figma](https://www.figma.com/design/NpI2wzW86XXPWxsSeomYfy/u.c%2Fcloud?node-id=31-1589&t=JDqOlKdc8TNDstzv-1)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12061](https://warthogs.atlassian.net/browse/WD-12061)

[WD-12061]: https://warthogs.atlassian.net/browse/WD-12061?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
